fe/fe-core/src/main/java/com/starrocks/mysql/MysqlProto.java

```diff
@@ -197,20 +197,36 @@ public static boolean negotiate(ConnectContext context) throws IOException {
             // 1. clear the serializer
             serializer.reset();
             // 2. build the auth switch request and send to the client
-            // TODO(yiming): support kerberos in new RBAC privilege framework later
```

Comment:
There are other two todo marks, plz check whether we need to modify those logic, if not, plz remove them directly.

![image](https://user-images.githubusercontent.com/107105845/223012376-e3f9a5f6-a9d2-4750-be2d-1e37f23d3ad7.png)